syft/execution/plan.py

```diff
@@ -582,42 +368,39 @@ def __str__(self):
-        out += "):\n"
-        for action in self.actions:
-            line = "    "
-            if action.return_ids is not None:
-                if isinstance(action.return_ids, PlaceHolder):
-                    tag = extract_tag(action.return_ids)
-                    line += f"_{tag} = "
-                elif isinstance(action.return_ids, tuple):
-                    line += (
-                        ", ".join(
-                            f"_{extract_tag(o)}" if isinstance(o, PlaceHolder) else str(o)
-                            for o in action.return_ids
-                        )
-                        + " = "
-                    )
-                else:
-                    line += str(action.return_ids) + " = "
-            if action.target is not None:
-                line += f"_{extract_tag(self.placeholders[action.target.value])}."
-            line += action.name + "("
-            line += ", ".join(
-                f"_{extract_tag(arg)}" if isinstance(arg, PlaceHolder) else str(arg)
-                for arg in action.args
-            )
-            if action.kwargs:
-                line += ", " + ", ".join(f"{k}={w}" for k, w in action.kwargs.items())
-            line += ")\n"
-            out += line
-
-        out += "    return "
-        out += ", ".join(f"_{extract_tag(p)}" for p in self.find_placeholders("output"))
+        # out += f"def {self.name}("
+        # out += ", ".join(f"arg_{extract_tag(p)}" for p in self.find_placeholders("input"))
+        # out += "):\n"
+        # for action in self.actions:
+        #     line = "    "
+        #     if action.return_ids is not None:
+        #         if isinstance(action.return_ids, PlaceHolder):
+        #             tag = extract_tag(action.return_ids)
+        #             line += f"_{tag} = "
+        #         elif isinstance(action.return_ids, tuple):
+        #             line += (
+        #                 ", ".join(
+        #                     f"_{extract_tag(o)}" if isinstance(o, PlaceHolder) else str(o)
+        #                     for o in action.return_ids
+        #                 )
+        #                 + " = "
+        #             )
+        #         else:
+        #             line += str(action.return_ids) + " = "
+        #     if action.target is not None:
+        #         line += f"_{extract_tag(self.placeholders[action.target.value])}."
+        #     line += action.name + "("
+        #     line += ", ".join(
+        #         f"_{extract_tag(arg)}" if isinstance(arg, PlaceHolder) else str(arg)
+        #         for arg in action.args
+        #     )
+        #     if action.kwargs:
+        #         line += ", " + ", ".join(f"{k}={w}" for k, w in action.kwargs.items())
+        #     line += ")\n"
+        #     out += line
+
+        # out += "    return "
+        # out += ", ".join(f"_{extract_tag(p)}" for p in self.find_placeholders("output"))
```

Comment:
Can we remove this now? 👍

Reply:
I want to because I don't find it very useful to print a list of actions like that... What do you think?